app/assets/stylesheets/components/_option-select.scss

```diff
@@ -157,6 +69,22 @@
     height: 5px;
     border-width: 0 0 3px 3px;
   }
+
+  .gem-c-checkboxes {
+    margin: 0;
+  }
+}
+
+.app-c-option-select__container-head {
+  padding: 0 5px;
+}
+
+.app-c-option-select__label {
+  @include core-19;
+  position: relative;
+  margin-right: 40px;
+  font-weight: bold;
```

Comment:
Font weight and `@inlude core-19` could be replaced with one line`@include govuk-font(48, $weight: bold);`